docs/graphql/manual/event-triggers/scheduled-triggers.rst

```diff
@@ -0,0 +1,126 @@
+
+Step 3: Define advanced options (Optional)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you like, you can also define the following values by expanding the ``Advanced`` section:
+
+- **Headers**: List of headers to be sent to the webhook.
+- **Retry configuration**: In case the call to the webhook fails.
+- **Include in metadata**: If a cron trigger is included in the metadata, you can export it when the metadata of the graphql-engine is exported.
+- **Comment**: Custom description of the cron trigger.
+
+.. thumbnail:: /img/graphql/manual/event-triggers/advanced-cron.png
+   :alt: Defining advanced options
+   :width: 809px
+
+Schedule & logs
+^^^^^^^^^^^^^^^
+
+Once you've created your cron trigger, you can see ``Pending events``, ``Processed events``, and ``Invocation logs`` in their respective tabs.
+
+.. thumbnail:: /img/graphql/manual/event-triggers/pending-cron.png
+   :alt: The tabs
+
+One-off scheduled events
+------------------------
+
+One-off scheduled events are used to reliably trigger a HTTP webhook to run custom business logic at a particular timestamp. For example, you can create a scheduled event to send a reminder email two weeks after a user signs up.
+
+You can schedule an event from your backend using the :ref:`metadata API <create_scheduled_event>`, or through the Hasura console.
+
+To add a one-off scheduled event via the console, follow these steps:
+
+Step 1: Navigate to One-off Scheduled Events
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Go to the ``Events`` tab in your Hasura console.
+- Click ``One-off Scheduled Events``.
+- Click ``Schedule an event``.
+
+.. thumbnail:: /img/graphql/manual/event-triggers/one-off.png
+   :alt: Adding a one-off scheduled event
+
+Step 2: Define the scheduled event
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the form opened by the above step, fill out the following fields:
+
+- **Webhook**: Enter the HTTP endpoint that should be triggered.
+- **Time**: Enter the time to trigger the event.
+- **Payload**: The JSON payload which will be sent to the webhook.
+
+.. thumbnail:: /img/graphql/manual/event-triggers/define-one-off.png
+   :alt: Defining the scheduled event
+   :width: 662px
+
+Step 3: Define advanced options (Optional)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you like, you can also define these values by expanding the ``Advanced`` section:
+
+- **Headers**: List of headers to be sent to the webhook.
+- **Retry configuration**: In case the call to the webhook fails.
+- **Comment**: Custom description of the scheduled trigger.
+
+.. thumbnail:: /img/graphql/manual/event-triggers/advanced-one-off.png
+   :alt: Defining advanced options
+   :width: 809px
+
+Schedule & logs
```

Comment:
Schedule & logs? Did you mean Scheduled events & logs?

Reply:
It means the schedule as in when the events get executed: https://deploy-preview-5141--hasura-docs.netlify.app/graphql/manual/event-triggers/scheduled-triggers.html#schedule-logs.